build.sh

```diff
@@ -18,6 +22,19 @@ while [[ "${1:-}" != "" ]]; do
         --skip-test|--skip-tests)
             runtarget="build"
             ;;
+        --scope)
```

Comment:
Can we default scope to be the package in the current directory in case this script is executed from a non-root directory?

Reply:
@eladb Great idea!